src/course-outline/hooks.jsx

```diff
@@ -112,15 +114,15 @@ const useCourseOutline = ({ courseId }) => {
   };
 
   const getUnitUrl = (locator) => {
-    if (getConfig().ENABLE_UNIT_PAGE === 'true') {
+    if (getConfig().ENABLE_UNIT_PAGE === 'true' || waffleFlags.useNewUnitPage) {
```

Comment:
Much as I dislike env vars, if we're keeping them, this should be an `&&`.

Reply:
[inform] Implemented in follow-up commit: [`5f19746` (#1372)](https://github.com/openedx/frontend-app-authoring/pull/1372/commits/5f19746fd6b2a803200abd8abdaac6adb5813fb2#diff-ee8befcc6ce4369bef43099a7729e264edac8a39043f7c184e0c30b743a5bb48R117)